packages/closed_household/pubspec.yaml

```diff
@@ -43,6 +43,11 @@ dependencies:
   registration_delivery: ^1.0.3
 
 
+dependency_overrides:
```

Comment:
This can be removed since there is local override file

Reply:
> This can be removed since there is local override file

okay